### DISCUSSION TITLE
Relationship of Eclipse-edc with Gaia-X

### DISCUSSION THREAD

**Discussion by MUsman1992** [eclipse-edc/.github#26] [General]
I have investigated the Gaia-X project and what I have understood is that both projects have some overlap. It seems that both projects want to achieve the same target. There are some services that have been developed in both projects like Federated Catalogue. I have a question. Can anyone tell me what is the exact relationship of the eclipse-edc with GAIA-X as in the GAIA-X project documentation, I have also seen that in GAIA-X ecosystem data exchange between the provider and the consumer will be done via connector which has been developed by eclispe-edc as can be seen in the following image. Are these 2 projects trying to achieve the same thing or will eclipse-edc provide some kind of components like connector that will be later used in GAIA-X ecosystem? Can someone guide me in this regard? 

![image](https://user-images.githubusercontent.com/62036653/236780652-9325b3ea-9a49-4314-a52d-216e952e14e5.png)

**Comment by Metalesaek** [comment]
The conception of the connector and all related services  are made by gaiax association following the rules of international dataspace IDS. Eclipce-edc, Fiiware, silicon, ... are all the technical implemention that follows the gaiax architecture and service
